### DISCUSSION TITLE
Add Void Color to Walls

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#16271] [Feature requests] [answered]
Adding Void Color to Walls doesn't seem like a difficult to implement feature...
![733737 Giga Park 2021-12-18 17-31-35](https://user-images.githubusercontent.com/94884086/146657231-107b6c36-ca43-402a-995f-7acb1b92b426.png)

**Comment by Xtreme97-RCT** [comment]
I've made a custom object to achieve this, not sure if this would be a good candidate to implement as an official object also?

I'll upload it here if you want to take a look:
[xtreme97.edge.void.zip](https://github.com/OpenRCT2/OpenRCT2/files/7740076/xtreme97.edge.void.zip)

**Reply by duncanspumpkin** [reply]
@Xtreme97-RCT could you submit it to official objects https://github.com/OpenRCT2/objects

**Comment by 733737** [comment, marked as answer]
I am loving this texture. Thanks so much, great improvement for my parks :)
